### PR TITLE
milestone5 :Views

### PR DESCRIPTION
**Pull request by abdessalem1998**
In this pull request, I need to review my code for adding the Views associated with the Models and Controllers already created for the Blog app. in this part I did the following:

### Project requirements
- Implement the design from the sneak peek wireframes.
    - Styling is nice to have but focus on making sure everything functions first.
   

- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- **Hint:** In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,21 @@
...
+    <% end %>
+    <% end %>
+  </div>
+  <div class="row d-block m-0 mt-5 mb-5 d-flex justify-content-center">
+    <button type="button" class="btn btn-secondary">
+      <%= link_to "See all", user_posts_path(user_id: @user.id) %>
+    </button>
+  </div>
+</div>
+</section>
```
Comment (julie-ify) [inline app/views/users/show.html.erb]: - [ ] On the users show page, each post should have the number of comments and likes under it as seen in the wireframe. Kindly add it.

chrome_CC9a1vca3z

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,21 @@
...
+      <% post.recent_comments.each do |comment| %>
+        <p class="m-2 border-bottom"><%= comment.author.name %>: <%= comment.text %></p>
+      <% end %>
+      <% end %>
+    </div>
+    <% end %>
+    <% end %>
+  </div>
+</div>
+</section>
```
Comment (julie-ify) [inline app/views/posts/index.html.erb]: - [ ] On the post index page, each post should have the number of comments and likes under it as seen in the wireframe. Kindly add it.

chrome_sU3KuUIl63

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -1 +1,18 @@
...
+      <% end %>
+    </div>
+    <div class="border m-2">
+      <% @post.comments.each do |comment| %>
+        <p class="m-2 border-bottom"><%= comment.author.name %>: <%= comment.text %></p>
+      <% end %>
+    </div>
+  </div>
+</div>
+</section>
```
Comment (julie-ify) [inline app/views/posts/show.html.erb]: - [ ] Kindly make this post show page to match the wireframe. As seen in the wireframe, the post author's name is at the top of the post beside the post title but yours is at the end. Kindly fix this
- [ ] Also add the number of comments and likes for the post as seen in the wireframe.

chrome_eRPLqB9GVS

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @abdessalem1998,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Good code organization and working app ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

- [ ] **[REQUIRED]** Kindly add the title of the project and a brief description of the project in the README to make your README more descriptive and informative.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @abdessalem1998,  

Your project is complete! 

## Good Points 👍 
- All pages are styled based on provided wireframe ✔️ 
- 3 recent posts and 5 recent comments functions are working ✔️ 
- No linter errors ✔️ 
- Professional documentation ✔️ 


giphy

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**